### DISCUSSION TITLE
Mask input validation

### DISCUSSION THREAD

**Discussion by clinvest** [alpinejs/alpine#3967] [7. Bugs (archived)] [answered]
Hi,

I'm not using the mask feature because it seems there is a bug. The filament team told me it's an alpine bug: https://github.com/filamentphp/filament/issues/10694

When I use the mask value, for example a number mask, in my field if I type "1111aaaa" it seems the aaaa is in the input value even if it is not visible in the input. As a result in filament: I cannot validate the field because in my field I see "1111" but the value of the input is "1111aaaa".

**Comment by ekwoka** [comment]
Does the resulting HTML use x-wire.model?

wire model is implemented in livewire, and has been the cause of some recent issues where mask seems to not be working.

**Comment by joshhanley** [comment, marked as answer]
@clinvest yeah @ekwoka is correct, this is a Livewire bug. There is a failing test PR open on the Livewire repo https://github.com/livewire/livewire/pull/7423
